api/monitoring/alerts.js

```
@@ -0,0 +1,476 @@
+    
+    const alertManager = getAlertManager();
+    
+    switch (req.method) {
+      case 'GET': {
+        // Get alert status and configuration
+        const { action } = req.query;
+        
+        switch (action) {
+          case 'status':
+            // Get current alert status
+            const status = alertManager.getStatistics();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)